test-clients/src/main/java/com/hedera/services/yahcli/commands/accounts/CreateCommand.java

```diff
@@ -67,9 +73,16 @@ public Integer call() throws Exception {
         final var effectiveMemo = memo != null ? memo : "";
         final var amount = SendCommand.validatedTinybars(yahcli, amountRepr, denomination);
         final var retries = boxedRetries != null ? boxedRetries.intValue() : DEFAULT_NUM_RETRIES;
+        final var effectiveReceiverSigRequired = receiverSigRequired;
```

Comment:
Why create a new variable here?

Reply:
just to make it consistent that it will not be able to changed.